### PR TITLE
Feature/disable check

### PR DESCRIPTION
**Pull request by absinth84**
Added a Flag (timeout = -2 ) to disable deploy check after service update.

### CODE REVIEW

**Comment by absinth84** [comment]
Why the automatic test with Travis-ci fails?

**Comment by fabfuel** [comment]
Hi @absinth84, this was related to the the new version 7.0.0 of Click.
I fixed this in the `develop` and `master` branches, please update your branch, then the build should pass.

File: README.rst [hunk 1]
```diff
@@ -264,6 +264,8 @@ The timeout error means, that AWS ECS takes longer for the full deployment cycle
 
 You can increase the time (in seconds) to wait for finishing the deployment via the ``--timeout`` parameter. This time includes the full cycle of stopping all old containers and (re)starting all new containers. Different stacks require different timeout values, the default is 300 seconds.
 
+If you set '--timeout = -2' check deployment is skipped
```
Comment (fabfuel) [inline README.rst]: This syntax is not correct, it needs to be `--timeout -2`

```suggestion
To completely skip the validation of the successful deployment, you can disable the timeout by setting::
    
    --timeout -2
```

File: ecs_deploy/cli.py [hunk 2]
```diff
@@ -239,14 +239,15 @@ def deploy_task_definition(deployment, task_definition, title, success_message,
...
 
-    wait_for_finish(
-        action=deployment,
-        timeout=timeout,
-        title=title,
-        success_message=success_message,
-        failure_message=failure_message,
-        ignore_warnings=ignore_warnings
-    )
+    if(timeout != -2):
```
Comment (fabfuel) [inline ecs_deploy/cli.py]: Why `-2`?

**Comment by fabfuel** [comment]
Hey, 
as there were no tests added yet, i implemented the "fire and forget" deployment in #81 (via `--timeout -1`).
Best, Fabian

**Comment by absinth84** [comment]
Hey Fabian,
Sorry for the late but i was in holiday..
Thank you!

I use ecs deploy in parallel mode from jenkins (15 - 20 services at the
same time x some clusters) and to not overcall the aws api and to not
receive to many call i added also a random sleep between 10 to 40 sec
before describe_services call.

Best,

Paolo Cerletti

Il giorno sab 9 feb 2019 alle ore 12:51 Fabian Fuelling <
notifications@github.com> ha scritto:

**Pull request closed without merging**